util/stringutil/string_util.go

```diff
@@ -241,12 +241,12 @@ func CompileLike2Regexp(str string) string {
 
 // DoMatchByte is a adapter for `DoMatchInner`, `str` can only be an ascii string.
 func DoMatchByte(str string, patChars, patTypes []byte) bool {
```

Comment:
DoMatchRune? or DoMatchBytes?

Reply:
updated. it should be `DoMatchBytes`, as the comment says this function should only accept an ascii string which one byte represents one character.